integration-tests/src/test/java/tachyon/client/TachyonFSIntegrationTestIso.java

```diff
@@ -234,11 +269,15 @@ public void unlockBlockTest2() throws IOException {
     for (int k = 0; k < numOfFiles; k ++) {
       tFile = mTfs.getFile(fileIds.get(k));
       Assert.assertTrue(tFile.isInMemory());
-      TachyonByteBuffer tBuf = tFile.readByteBuffer(0);
-      Assert.assertNotNull(tBuf);
-      tBuf = tFile.readByteBuffer(0);
-      Assert.assertNotNull(tBuf);
-      tBuf.close();
+      is = tFile.getInStream(ReadType.CACHE);
+      buf = ByteBuffer.allocate((int) tFile.getBlockSizeByte());
+      int r = is.read(buf.array());
+      Assert.assertTrue(r != -1);
+      is.seek(0);
+      buf.clear();
+      r = is.read(buf.array());
+      Assert.assertNotNull(r != -1);
```

Comment:
This can be combined with the previous line.

This should be assertTrue(), not assertNotNull().